packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/formatMessage.js

```diff
@@ -29,8 +29,14 @@ const actionTypes = {
   'permission.delete': 'Delete permission',
 };
 
-const getDefaultMessage = (value) => {
+export const getDefaultMessage = (value) => {
   return actionTypes[value] || value;
 };
 
-export default getDefaultMessage;
+export const getActionMessage = (message, action) => {
+  if (message.includes('entry')) {
+    return `${message} (${action.model})`;
+  }
+
+  return message;
```

Comment:
Tbh I am not a fan of this. Can we handle this as part of the translation file, because that way we don't impose an order that might be different in other languages on the translation.

Reply:
ok works for me, I'll take a look at the react-intl select